nomad/structs/diff_test.go

```diff
@@ -4486,6 +4486,12 @@ func TestTaskDiff(t *testing.T) {
 						Type: DiffTypeAdded,
 						Name: "Artifact",
 						Fields: []*FieldDiff{
+							{
+								Type: DiffTypeAdded,
+								Name: "ForceDownload",
+								Old:  "",
+								New:  "false",
+							},
```

Comment:
A better test would be to flip the field, like `true -> false`, or from not declared to declared. Since there are two artifacts you could one one case on each.